### PR TITLE
Add support for multiline PR comments

### LINKED ISSUE #1297: Support multi-line comments with custom media header
GitHub Developer API announcement:
https://developer.github.com/changes/2019-10-03-multi-line-comments/

This would be a great PR for any new contributor to this repo or a new Go developer.
All contributions are greatly appreciated!

Feel free to volunteer for any issue and the issue can be assigned to you so that others don't attempt to duplicate the work.

Please check out our [CONTRIBUTING.md](https://github.com/google/go-github/blob/master/CONTRIBUTING.md) guide to get started.

Thank you!

### PR DESCRIPTION
**Pull request by davidlopezre**
- Fix #1297

### CODE REVIEW

**Comment by davidlopezre** [comment]
@googlebot I signed it!

**Comment by dalzymendoza** [comment]
@googlebot I fixed it.

**Comment by dalzymendoza** [comment]
@googlebot I consent.

File: github/github_test.go [hunk 1]
```diff
@@ -111,9 +111,14 @@ func testFormValues(t *testing.T, r *http.Request, values values) {
 }
 
 func testHeader(t *testing.T, r *http.Request, header string, want string) {
-	if got := r.Header.Get(header); got != want {
-		t.Errorf("Header.Get(%q) returned %q, want %q", header, got, want)
+	relevantHeaders := r.Header[header]
```
Comment (gmlewis) [inline github/github_test.go]: Sorry, @davidlopezre - this change should be reverted, as it is not correct.  See below.

File: github/pulls_comments.go [hunk 2]
```diff
@@ -79,6 +81,7 @@ func (s *PullRequestsService) ListComments(ctx context.Context, owner string, re
 
 	// TODO: remove custom Accept header when this API fully launches.
 	req.Header.Set("Accept", mediaTypeReactionsPreview)
+	req.Header.Add("Accept", mediaTypeMultiLineCommentsPreview)
```
Comment (gmlewis) [inline github/pulls_comments.go]: This is not how we set the Accept header when multiple fields are needed. Here is an example:

```
	// TODO: remove custom Accept header when this API fully launches.
	acceptHeaders := []string{mediaTypeLabelDescriptionSearchPreview, mediaTypeLockReasonPreview, mediaTypeDraftPreview}
	req.Header.Set("Accept", strings.Join(acceptHeaders, ", "))
```

**Comment by gmlewis** [comment]
Reopening per #1297 conversation.

**Comment by davidlopezre** [comment]
Hi @gmlewis,

Sorry for the delay in fixing this PR. I have addressed your comments.

Regarding the way we were adding the `Accept` headers, I have a question. Why is it preferable to concatenate them in a single key? I tried to look it up in the HTTP specification but I did not find anything related.

**Comment by gmlewis** [comment]
> Regarding the way we were adding the Accept headers, I have a question. Why is it preferable to concatenate them in a single key? I tried to look it up in the HTTP specification but I did not find anything related.

All I can tell you is that we experimented several ways with the GitHub v3 API endpoints, and this mechanism is the only way we got it to work.

**Comment by gmlewis** [comment]
Thank you, @wesleimp!
Merging.

**Pull request merged**